### ISSUE TITLE
Losing car upgrades due to garages not storing vehicles properly/cars poofing before storing/cars not being able to be stored.

### ISSUE THREAD

**Issue by BlownOut734** [InZidiuZ/legacy-assets-issue-tracker#177]
**Reporter:**
Who is reporting this? Please include discord name.
crucifried#6758
**Summary:**
Write a summary of what you're experiencing.
Losing upgrades on a car right after doing them. I did a few upgrades to my 180sx such as bumpers/skirts and turbo. I left customs and stored my vehicle, pulled it out, went and got more money, then went back to customs and purchased engine 4, transmission, a new hood, and exhaust. When I left, I went back to my house to store my vehicle, it allowed me to but when I went to pull it out it said it was "Out" thus making me have to retrieve it and making all of the upgrades I just did to the car vanish. As if I didn't just spend $15k.
## Additional Information
I just think its extremely bad when we are in the city for x amount of time just to LITERALLY get our money wasted like we weren't working for it or something. I myself don't make money like that so yeah it does legitimately hurt my pockets. 
**Screenshots:**
If suitable, include some screenshots.

**Notes:**
For anything else noteworthy.
Just wish this wasn't a thing, it's a lot of money yanno? :/

**Comment by iNubby** [comment]
I've had this happen to me several times where constant money is just lost from bugs and isn't able to be compensated, which is annoying considering the amount of time it takes for some people to make that amount of money from the job they choose to do. I've lost thousands in upgrades from my car poofing while driving it, the garage not saving the upgrade at all, or just not being able to store the vehicle. For example, I used to own an Elegy Retro Custom and purchased a $5500 spoiler for it, I put it in the garage and I got off, came back on the next day for my car to not have the spoiler at all and was told to make the money back and rebuy it and that I won't be able to be compensated for it.
